packages/react-storage/src/components/StorageManager/StorageManager.tsx

```diff
@@ -43,13 +49,41 @@ function StorageManagerBase(
     processFile,
     components,
     path,
+    prefix,
   }: StorageManagerProps,
   ref: React.ForwardedRef<StorageManagerHandle>
 ): JSX.Element {
-  if (!accessLevel || !maxFileCount) {
-    logger.warn('StorageManager requires accessLevel and maxFileCount props');
+  if (!maxFileCount) {
+    logger.warn(
+      '`StorageManager` requires the `maxFileCount` prop to be specified'
+    );
+  }
+
+  if (!prefix && !accessLevel) {
```

Comment:
Rather than throwing here, we should set accessLevel to private, which is the current default.

Reply:
Discussed offline. We should not throw an error in this case because of the current default